esp/templates/program/manage_programs.html

```diff
@@ -34,6 +34,8 @@ <h2>Hello Admins!</h2>
           <li><a href="/manage/statistics/">View statistics about students</a></li>
           
           <li><a href="/manage/emails/">Monitor comm panel email status</a></li>
+          
+          <li><a href="/manage/tags/">Modify tag settings (generic miscellaneous settings, for experts only)</a></li>
```

Comment:
I feel like this (and some of the other links as well) should exist outside of a QSD that could get erroneously editted out/not updated/etc. Could we make a more official and permanent link somewhere?

Reply:
I've been feeling the same way for quite a while, now (in fact, since we had to go through and manually change everyone's QSDs on this page during the last stable release). Really, we probably want everything in this QSD to be part of the template, then have a QSD below it for chapters to customize.